homeassistant/helpers/entity_registry.py

```diff
@@ -703,6 +706,19 @@ async def async_load(self) -> None:
 
         from .entity import EntityCategory  # pylint: disable=import-outside-toplevel
 
+        def parse_enum(
+            cls: type[_StrEnumT], value: str | None, default: _StrEnumT | None
+        ) -> _StrEnumT | None:
+            """Parse enum and ignore invalid values."""
+
+            if not value:
+                return None
+            try:
+                return cls(value)
+            except ValueError:
+                _LOGGER.warning("Ignoring unexpected value %s for enum %s", value, cls)
```

Comment:
I don't think we should just ignore every unknown value.

Reply:
Breaking startup of home assistant due to a single errounous value here is not a good ide.

Reply:
An alternative would be to  ignore the complete entity with bad data.

Reply:
> Breaking startup of home assistant due to a single errounous value here is not a good ide.

There are thousands of ways custom integrations could break the registries 🤷

Reply:
Nope, we have two reported issues that i know of so far.